data.sql

```diff
@@ -1,14 +1,64 @@
 /* Populate database with sample data. */
 
 INSERT INTO public.animals(
-	id, name, date_of_birth, escape_attempts, neutered, weight_kg)
-	VALUES (1, 'Agumon', '03-02-2020', 0, true, 10.23);
+	name, date_of_birth, escape_attempts, neutered, weight_kg)
+	VALUES ('Agumon', '03-02-2020', 0, true, 10.23);
 INSERT INTO public.animals(
-	id, name, date_of_birth, escape_attempts, neutered, weight_kg)
-	VALUES (2, 'Gabumon', '15-11-2018', 2, true, 8);
+	name, date_of_birth, escape_attempts, neutered, weight_kg)
+	VALUES ('Gabumon', '15-11-2018', 2, true, 8);
 INSERT INTO public.animals(
-	id, name, date_of_birth, escape_attempts, neutered, weight_kg)
-	VALUES (3, 'Pikachu', '07-01-2021', 1, false, 15.04);
+	name, date_of_birth, escape_attempts, neutered, weight_kg)
+	VALUES ('Pikachu', '07-01-2021', 1, false, 15.04);
 INSERT INTO public.animals(
-	id, name, date_of_birth, escape_attempts, neutered, weight_kg)
-	VALUES (4, 'Devimon', '12-05-2017', 5, true, 11);
+	name, date_of_birth, escape_attempts, neutered, weight_kg)
+	VALUES ('Devimon', '12-05-2017', 5, true, 11);
+INSERT INTO public.animals(
+	name, date_of_birth, escape_attempts, neutered, weight_kg)
+	VALUES ('Charmander', '2020-02-08', 0, false, -11);
+INSERT INTO public.animals(
+	name, date_of_birth, escape_attempts, neutered, weight_kg)
+	VALUES ('Plantmon', '2021-11-15', 2, true, -5.7);
+INSERT INTO public.animals(
+	name, date_of_birth, escape_attempts, neutered, weight_kg)
+	VALUES ('Squirtle', '1993-04-02', 3, false, -12.13);
+INSERT INTO public.animals(
+	name, date_of_birth, escape_attempts, neutered, weight_kg)
+	VALUES ('Angemon', '2005-06-12', 1, true, -45);
+INSERT INTO public.animals(
+	name, date_of_birth, escape_attempts, neutered, weight_kg)
+	VALUES ('Boarmon', '2005-06-07', 7, true, 20.4);
+INSERT INTO public.animals(
+	name, date_of_birth, escape_attempts, neutered, weight_kg)
```

Comment:
- [ ] Kindly place all the values in `' '` inverted commas. Example-: 
```
INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
    VALUES ('Plantmon', '2022/11/15', '2', 'true', '-5.7');
```
Make sure to make changes for all animals.